### PR TITLE
[Parallel] Fix missing --xdebug in WorkerCommand line on WorkerCommandLineFactory when --xdebug provided

### LINKED ISSUE #8060: Xdebug doesn't work within custom rule's `refactor` method.
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | 0.14.4, 0.17.6 |

I am trying to create a custom rule, in which the xdebug breakpoint does get triggered in my custom rule's: `__construct` method, but not in the `refactor(...)` method.

## Minimal PHP Code Causing Issue

```
final class CustomRule extends AbstractRector
{
    public function __construct() {
         dump("__construct"); // This breakpoint triggers
    }

    /**
     * @return array
     * <class-string
     * <Node>>
     */
    public function getNodeTypes(): array
    {
        // what node types are we looking for?
        // pick any node from https://github.com/rectorphp/php-parser-nodes-docs/
        return [Node\Attribute::class];
    }

    /** @param Node\Attribute $node */
    public function refactor(Node $node): ?Node
    {
        dump("refactor()"); // This breakpoint does *not* trigger
        return $node;
    }
```

## Expected Behaviour

I would expect the breakpoint to be hit and execution paused.

### PR DESCRIPTION
**Pull request by samsonasik**
@zangai @deviantintegral @MartinsRucevskis @staabm  could you verify manually this patch on  `WorkerCommandLineFactory` can fix https://github.com/rectorphp/rector/issues/8060

Thank you.

### CODE REVIEW

**Comment by samsonasik** [comment]
I see, the issue was it missing register option to `ProcessConfigureDecorator`, I added it https://github.com/rectorphp/rector-src/pull/5398/commits/4786e34835e73f527b08b6a8f406e1eb724dd85f

**Comment by samsonasik** [comment]
@TomasVotruba I added unit test on `WorkerCommandLineFactoryTest` for it https://github.com/rectorphp/rector-src/pull/5398/commits/91de44cb79cbbb95efdce2be4cf26e15215cc276

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: src/Console/ConsoleApplication.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function doRun(InputInterface $input, OutputInterface $output): int
     {
         // @fixes https://github.com/rectorphp/rector/issues/2205
         $isXdebugAllowed = $input->hasParameterOption('--xdebug');
-        if (! $isXdebugAllowed) {
+        if ($isXdebugAllowed) {
```
Comment (samsonasik) [inline src/Console/ConsoleApplication.php:46]: by register option xdebug to `ProcessConfigureDecorator`, the is xdebug has parameter option seems can use truthy check 👍

**Comment by TomasVotruba** [comment]
Let's ship it :+1:

**Comment by staabm** [comment]
in phpstan reecntly the internal worker commands have been marked as hidden, so they don't confuse the end user

https://github.com/phpstan/phpstan-src/pull/2833

**Pull request merged**